tests/images.spec.ts

```diff
@@ -18,3 +28,60 @@ test("search for custom image and create an instance from it", async ({
 
   await deleteInstance(page, customInstance);
 });
+
+test("Export an image", async ({ page }) => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+  const imageAlias = await createImage(page, instance);
+
+  await page.goto("/ui/");
+  await page.getByRole("link", { name: "Images", exact: true }).click();
+  await page.getByPlaceholder("Search").click();
+  await page.getByPlaceholder("Search").fill(imageAlias);
+  page.getByText(imageAlias);
+  const imageName = await getImageNameFromAlias(page, imageAlias);
+  await page.getByLabel("export image").click();
+  await page.waitForSelector(
+    `text=Image ${imageName} was successfully downloaded. Please check your downloads folder.`,
+  );
+
+  await deleteImage(page, imageName);
+  await deleteInstance(page, instance);
+});
+
+test("Upload an image", async ({ page }) => {
+  const IMAGE_FILE = "./tests/fixtures/e2e-test-image.tar.gz";
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+  await visitInstance(page, instance);
+  const instanceBaseImageName = await page
+    .locator("text=Base image")
+    .locator("xpath=following-sibling::*")
+    .innerText();
```

Comment:
```suggestion
```
I think we don't need the instance or the image name for this test?

Reply:
I see we need to create an instance. Because that will download an image. Because we deleted all images previously with another test. Maybe don't call delete all images in other tests that operate on the default project. Then we don't need an instance for this one.

Reply:
I think you are right. I have recreated this function so it is much more simplified.